waku/incentivization/rpc.nim

```diff
@@ -0,0 +1,17 @@
+import
+  json_serialization,
+  std/options
+import
+  ../waku_core
+
+# Exactly following the RFC:
+# https://github.com/vacp2p/rfc/tree/master/content/docs/rfcs/73
+
+type
+
+  EligibilityProof* = object
+    proof*: Option[seq[byte]]
+
+  EligibilityStatus* = object
+    statusCode*: Option[uint32]
```

Comment:
I think this should be a mandatory field, so should not be an `Option`. See https://github.com/waku-org/nwaku/blob/master/waku/waku_filter_v2/rpc.nim#L23-L26 for analogous implementation for Filter. Note that in the RFC protobuf this field must be marked `optional`. This is protobuf weirdness and orthogonal to whether we consider this field mandatory or not.